lessons/module-3/react-2-the-how.md

```diff
@@ -253,47 +236,40 @@ Failed to compile.
 ./src/App.js
   Line 12:  Parsing error: Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>?
 
-  10 |     return(
-  11 |       <h1>IdeaBox</h1>
-> 12 |       <p>Hi!</p>
+  12 |     return(
+  13 |       <h1>IdeaBox</h1>
+> 14 |       <p>Hi!</p>
      |       ^
-  13 |     )
-  14 |   }
-  15 |
+  16 |     )
+  17 |   
+  18 |
 ```
 
 Let's figure out what this error is saying. "Adjacent JSX elements must be wrapped in an enclosing tag."
 
-If we think about this logically, it makes sense! The method `render` is just a function - a regular old class method. And how many things can a function return at once? Just one! So in order to return multiple JSX elements, we have to wrap them in a single JSX element!
+If we think about this logically, it makes sense! Our component is just a function. And how many things can a function return at once? Just one! So in order to return multiple JSX elements, we have to wrap them in a single JSX element!
 
 Since this is our App component, let's wrap everything in a `<main>` tag!
 
 <section class="note">
 ### NOTE  
 
-If you're just looking for an unflavored container for your elements (aka they are not semantically related to one another, and the best element to use would be a `<div>`), then instead, use a `<React.Fragment>`! You can read more about Fragments [here](https://reactjs.org/docs/fragments.html).
+If you're just looking for an unflavored container for your elements (aka they are not semantically related to one another, and the best element to use would be a `<div>`), then instead, use a `<React.Fragment>`! You can read more about Fragments [here](https://react.dev/reference/react/Fragment).
 </section>
 
 ```js
-import React, { Component } from 'react';
+import  {useState} from 'react';
```

Comment:
Similar to Leta's comment before, I might add the spacing between the curly braces just to be consistent.